### PR TITLE
feat: move code into new repository

### PR DESCRIPTION
**Pull request by Thund3rHawk**

### CODE REVIEW

**Comment by Thund3rHawk** [comment]
Hello @Tpuljak, please check.

File: README.md [hunk 1]
```diff
@@ -1,34 +1,251 @@
...
+## 🚀 Getting Started
 
-## ✨ Features  
+### Open Using Daytona
+
+1. **Install Daytona**: Follow the [Daytona installation guide](https://www.daytona.io/docs/installation/installation/).
+2. **Create the Workspace**:
+
+   ```bash
+   daytona create https://github.com/Thund3rHawk/fitness-assistant.git
```
Comment (Tpuljak) [inline README.md]: Link is wrong here

**Review by Tpuljak — CHANGES_REQUESTED** [review]
Please also sign your commit.

I suggest squashing everything into one commit and signing that one

File: README.md (hunk 1 above)
Comment (Thund3rHawk) [reply README.md]: @Tpuljak updated.

**Comment by Thund3rHawk** [comment]
@Tpuljak just updated.

**Comment by mojafa** [comment]
@Thund3rHawk please sign your commits. To amend your last commit to include the sign-off you can do  this:
`gc -a -s -m "commit message”` then `git push --force`

**Comment by Thund3rHawk** [comment]
@mojafa sign-off done.

File: README.md (hunk 1 above)
Comment (Tpuljak) [reply README.md]: The link is still wrong

File: README.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ The source code for this project can be found at the following GitHub repository
 2. **Create the Workspace**:
 
    ```bash
-   daytona create https://github.com/Thund3rHawk/fitness-assistant.git
+   daytona create https://www.daytona.io/docs/installation/installation/
```
Comment (Tpuljak) [inline README.md]: THe URL needs to be of this repo. (https://github.com/daytonaio/sample-typescript-gemini-fitness-assitant)

**Review by Tpuljak — CHANGES_REQUESTED** [review]

**Comment by mojafa** [comment]
@Thund3rHawk still fialing DCO. Sorry please run :
```
git rebase HEAD~41 --signoff
git push --force-with-lease origin main

```

**Comment by Tpuljak** [comment]
@mojafa all good, I'll squash the commits when I merge. We need at least one commit signed

**Comment by Thund3rHawk** [comment]
@mojafa @Tpuljak sign-off done

**Pull request merged**